api/src/routers/last-meal.js

```diff
@@ -0,0 +1,10 @@
+import express from "express";
+import knex from "../database_client.js";
+
+
+const lastMeal = express.Router();
+lastMeal.get('/', async(req , res)=> {
+    const last_meal = await knex('meal').orderBy('id', 'desc').limit(1)
```

Comment:
Currently, this API returns 200 (ok) when the last meal is not available, but the requirement for this API is a 404 (not found) response with a message.

> /first-meal and /last-meal should in that case return a 404 response with an explanation that there are no meals